org.adempiere.base/src/org/compiere/print/PrintUtil.java

```diff
@@ -489,7 +490,7 @@ public static void setupPrintForm (int AD_Client_ID, String trxName)
 			UUIDGenerator.updateUUID(MColumn.get(ctx, X_AD_PrintForm.Table_Name, PO.getUUIDColumnName(X_AD_PrintForm.Table_Name)), trxName);
 		
 		//
-		CLogMgt.enable(true);
+		CLogMgt.setLevel(level);

```

Comment:
this workaround the issue but doesn't fix the bug with the enable method.

Reply:
yeah, i'm aware with the error. i check that the problem that when copy value PO, there is change on value that being copied. need more inspection.